python/src/MaximumDistribution_doc.i.in

```diff
@@ -0,0 +1,62 @@
+%feature("docstring") OT::MaximumDistribution
+"Maximum distribution.
+
+Available constructors:
+    MaximumDistribution(*distribution*)
+
+    MaximumDistribution(*collection*)
+
+    MaximumDistribution(*distribution, size*)
+
+Parameters
+----------
+distribution : :class:`~openturns.Distribution`
+    The underlying distribution.
+collection : sequence of :class:`~openturns.Distribution`
+    A collection of pdfs.
+size : int
+    Number of instances of distribution.
+
+Notes
+-----
+The maximum distribution of F is the distribution of :math:`X = max(X_1, ... , X_n)`
+where :math:`(X_1, ... , X_n) \tilde F`
+
+.. math::
+
+    P(X < x) = P(X_1 < x \wedge ... \wedge X_n < x)
```

Comment:
You should use the macros defined in math_notations and use \leq instead of <:
\Prob{X\leq x}=\Prob{X_1\leq x,\dots,X_n\leq x}
It simplifies into \Prob{X\leq x}=\prod_{i=1}^n F_i(x) when X_1,\dots,X_n are independent (second constructor) and finaly it simplifies into F^n(x) when the random variables are iid (third constructor).